src/v2/controllers/FormController.js

```diff
@@ -46,6 +48,22 @@ export default Controller.extend({
       currentViewState.name,
       this.options.appState.get('authenticatorKey'),
     );
+
+    // Walk through every authenticator and apply customizations
+    this.options.appState
+      .getRemediationAuthenticationOptions(currentViewState.name)
+      .forEach(authenticator => {
+        // Open question:
+        // Should we hardcode a list or rely on our API responses?
+        const classIconOverrides = ['mfa-hotp'];
```

Comment:
Am leaning toward removing this hardcoded value in favor of relying on the IDX response to know when we have a `logo` field.

This allows us to extend support to other authenticators (IdP, OTP, etc.) without a Widget release.